connectors/aws-cert/src/test/java/com/vmware/connectors/aws/cert/AwsCertControllerTests.java

```diff
@@ -54,7 +54,6 @@ public void setup() throws Exception {
     @Test
     public void testProtectedResource() throws Exception {
         testProtectedResource(POST, "/cards/requests");
-        testProtectedResource(GET, "/success?code=test-auth-code");
```

Comment:
This was a leftover from ServiceNow that I never noticed and copy-pasted into a few tests.